src/components/Budget/BudgetView/BudgetView.js

```diff
@@ -0,0 +1,75 @@
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  AccordionSet,
+  Accordion,
+  Col,
+  Row,
+  ExpandAllButton,
+} from '@folio/stripes/components';
+import { ViewMetaData } from '@folio/stripes/smart-components';
+
+import {
+  expandAll,
```

Comment:
let's try to implement hooks for these funcs

Reply:
Added [hook](https://github.com/folio-org/stripes-acq-components/pull/43)